File: Python Word Examples/BackgroundApiDemo/deleteBackground.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.background_api.BackgroundApi(configuration)

filename = 'background.docx'
dest_file_path = 'output/DeleteBackground.docx'
extra = {'folder': "input",
         'dest_file_path': dest_file_path}
result = api.delete_background(filename, **extra)

File: PHP Word Examples/BackgroundApiDemo/deleteBackground.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BackgroundApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BackgroundApi($configuration);

$name = "deleteBackground.docx";
$password = null;
$folder = "input";
$storage = null;
$dest_file_path = "output/deleteBackground.docx";
$result=$apiInstance->deleteBackground($name,$dest_file_path, $password, $folder, $storage);